pkg/driver_cryptocell/Makefile

```diff
@@ -0,0 +1,10 @@
+PKG_NAME=driver_cryptocell
+PKG_URL=https://github.com/Einhornhool/RIOT-cryptocell-driver.git
+PKG_VERSION=24ff1f69ce77bd69ae1d1011daa1a2d238793f23
```

Comment:
Is there any reason to depends on your repo ?
The nordic library seems to be available [here](https://github.com/nrfconnect/sdk-nrfxlib/tree/main/crypto/nrf_cc310_bl/lib/cortex-m4/hard-float)

Reply:
Yes, but that version only provides header files for SHA-256 and ECDSA with a P-256 curve.
The version I include here provides headers for all available operations.

Reply:
i found a [nrf_cc310_bl_ecdsa_verify_secp256r1.h](https://github.com/nrfconnect/sdk-nrfxlib/blob/v2/crypto/nrf_cc310_bl/include/nrf_cc310_bl_ecdsa_verify_secp256r1.h)
seems like ECDSA with a P-256 to me

but i am not sure if that is the required header

would be nice if you provide a link to the upstream you used in your repository

Reply:
Yes, as I said, it provides ECDSA with a P-256 curve and SHA-256. The version I provide supports much more (ciphers, other hashes, MAC operations, multiple curves, etc.)

I originally got it from the Nordic Website. To get it from there I'll have to download a zip-file of the whole Nordic SDK and extract the archive and the header files from it. I'll see if that's possible.

Reply:
I updated the Makefile.
Now it downloads the Nordic SDK as a zip file, extracts the header files and the archive and uses them to build the package.